Polishing Dashboard page

pva701 opened the pull request:
## Description

<!--
Describes the nature of your changes. If they are substantial, you should
further subdivide this into a section describing the problem you are solving and
another describing your solution.
-->

## Related issue(s)

<!--
- Short description of how the PR relates to the issue, including an issue link.
For example
- Fixed #100500 by adding lenses to exported items

Write 'None' if there are no related issues (which is discouraged).
-->

https://issues.serokell.io/issue/EDNA-80
https://issues.serokell.io/issue/EDNA-88

## :white_check_mark: Checklist for your Pull Request

<!--
Ideally a PR has all of the checkmarks set.

If something in this list is irrelevant to your PR, you should still set this
checkmark indicating that you are sure it is dealt with (be that by irrelevance).

If you don't set a checkmark (e. g. don't add a test for new functionality),
you must be able to justify that.
-->

#### Related changes (conditional)

- Tests
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - [ ] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)

#### Stylistic guide (mandatory)

- [ ] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Commit-and-PR-policy-4cf98e1a910a415d86b5f2491d9af1af).
- [ ] My code complies with the [style guide](../tree/master/docs/code-style.md).

pva701 committed:
Reset new sub experiment information on Dashboard leave

pva701 committed:
Implement ContextActions for experiment

Metadata works for now, download not yet.

pva701 committed:
Context menu of subexperiment

pva701 committed:
Fix several bugs

Plot is not shrinking now on new subexperiment creation.
Default cursor in subexperiments, instead of textual.
Ellipsis too long subexperiment names.

pva701 committed:
Refresh selected subexperiments if a selector changes

pva701 committed:
Make possible mark experiment as suspicious

pva701 committed:
Add Tooltip component and use it where possible

pva701 committed:
Make possible to download excel file of experiment

pva701 committed:
Make new experiment plate collapsible

pva701 committed:
Show all experiments when all subexperiments deactivated

gromakovsky commented on frontend/src/pages/dashboard/ExperimentsTable/ExperimentsTable.tsx:
```diff
@@ -126,10 +139,41 @@ export function ExperimentsTableSuspendable({
+                  const metadata = await cacheMetadata(e.id);
+                  setModalDialog({
+                    kind: "show-experiment-metadata",
+                    description: metadata.fileMetadata.concat(metadata.description),
```
I see that now description is shown last, i. e. after all other metadata. I think description is likely to be more important than `fileMetadata` and it's better to show it first, but I am not entirely sure so it's up to you to decide.

gromakovsky commented:
Found something weird. I don't know what's the source of this bug, but maybe it's introduced in this PR, in this case please fix it.

![image](https://user-images.githubusercontent.com/2556409/115187446-0023bc00-a0ec-11eb-9967-8690146f8ab3.png)

gromakovsky commented:
According to https://serokell.slack.com/archives/C01MH299XEU/p1618811108000800, names of primary sub-experiments should be in **bold**. Can you please implement it here?

pva701 committed:
Make margin space smaller for metadata lines

Problem: now metadata looks too wide.

Solution: decrease <p> margins. Also swap
file metadata and manually written metadata.

pva701 committed:
Make primary subexperiment bold

Problem: it doesn't correspond to the design.

Make a primary subexperiment bold.

gromakovsky reviewed (approved):
My concerns seem to be fixed.

sashasashasasha151 commented:
![image](https://user-images.githubusercontent.com/25207759/115362961-b65dd300-a1ca-11eb-92b5-153d64fd2691.png)
Don't see error on invalid sub-experiments

sashasashasasha151 commented:
![image](https://user-images.githubusercontent.com/25207759/115363201-ed33e900-a1ca-11eb-8dab-8070fe2b5ee8.png)
Plot is not in full height

sashasashasasha151 commented:
![image](https://user-images.githubusercontent.com/25207759/115363326-09378a80-a1cb-11eb-802e-74e7f8113d5e.png)
No margin in the bottom

sashasashasasha151 commented on frontend/src/components/DialogLayout/DialogLayout.scss:
```diff
@@ -77,6 +77,11 @@
   border-radius: 6 * $px;
 
+  p {
+    margin-top: 2 * $px;
+    margin-bottom: 2 * $px;
```
Still don't understand why you like splitting it

gromakovsky commented:
I'll create a few issues to follow-up. I want to merge it now because we are going to have a demo today.

gromakovsky commented:
New issues regarding dashboard frontend:
* https://issues.serokell.io/issue/EDNA-99
* https://issues.serokell.io/issue/EDNA-100
* https://issues.serokell.io/issue/EDNA-102
